frontend/messages/nl.json

```
@@ -925,6 +925,8 @@
 	"taskScheduleWarning": "Alle taken die niet deel uitmaken van het nieuwe schema zullen permanent worden verwijderd",
 	"mappingSuggested": "Suggestie voor in kaart brengen",
 	"mappingSuggestedHelpText": "Suggesties voor in kaart brengen gebaseerd op geladen kaders",
+	"mappingSuggestionTeasing": "Suggesties voor het in kaart brengen zijn nu beschikbaar! U kunt de filters gebruiken om suggesties te verkennen op basis van geladen kaders.",
```

Biome (1.9.4)

[error] The key mappingSuggestionTeasing was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)